src/Microsoft.Health.SqlServer/Features/Schema/SchemaJobWorker.cs

```diff
@@ -60,11 +60,12 @@ public async Task ExecuteAsync(SchemaInformation schemaInformation, string insta
                         int? previous = schemaInformation.Current;
                         schemaInformation.Current = await schemaDataStore.UpsertInstanceSchemaInformationAsync(instanceName, schemaInformation, cancellationToken);
 
-                        // If there was a change in the schema version
-                        if (previous != schemaInformation.Current)
+                        // If there was a change in the schema version and this isn't the base schema
+                        if (schemaInformation.Current != previous && schemaInformation.Current > 0)
                         {
-                            // Fire a notification
-                            await _mediator.NotifySchemaUpgradedAsync((int)schemaInformation.Current);
+                            var isFullSchemaSnapshot = previous == null;
```

Comment:
While going through the code to ensure that changes from [this commit](https://github.com/microsoft/healthcare-shared-components/commit/6e0eb86d88a6ec261cb43f4de841039f764274cb) are re-added in this PR, I caught a bug in my code: this should really be `isFullSchemaSnapshot = previous == 0`.

I understand that this is somewhat unrelated to your changes, but if you could fix that here that would be great (if not, np! I am happy to create a bug).

Reply:
I have made the fix and updated the PR.